### DISCUSSION TITLE
Equipo 7 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#13] [Proyecto Final]
Hola @sowisla 
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola Sofía, @sowisla 

Gracias por entregar la etapa 0 del proyecto final. Escribo con retroalimentación, espero te ayude en la preparación para tu entrevista esta semana. 

En primer lugar, me parece un asunto muy importante, que impacta a muchas personas al momento de terminar su carrera. Me gusta demasiado el nombre del equipo/proyecto. Las preguntas que has creado son apropiadas y bien construidas. Algunas sugerencias: añadir una pregunta (or broma o historia) para romper el hielo, recoger información sobre la carrera de la persona, si el/ella tiene trabajo ahora y cuando logró recibir empleo, que tipo de trabajo es, etc. Es decir, información sobre la persona para que podrías interpretar sus respuestas dentro de la carrera que tiene. Y sería importante avisar a la entrevistado/a sobre la duración de la entrevista _antes_ de empezarla, para que la persona esté preparada.

¿Has elegido a una entrevistada? Mi recomendación sería elegir a alguien tan pronto como sea posible, agendar y completar la entrevista y sintetizar las notas según los vídeos en la plataforma sobre mapas de empatía/afinidad/viaje. Solo que necesario crear una mapa.

¿Tienes preguntas para mí? Cualquier cosa, me avisas.

¡Adelante!
Saludos,
Kasturi

**Reply by sowisla** [reply]
¡Muchas gracias por la retroalimentación, @KasturiSShah!

**Comment by KasturiSShah** [comment]
Hola @sowisla 

Gracias por entregar la etapa 1 del proyecto final. He revisado tu entrega y aprendí mucho sobre los temas importantes de tu entrevista, aun cuando no estuve presente durante ella. Significa que organizaste de manera eficiente la información recogida y que la presentación de información (orden de la agrupación de temas, colores, etc) fue apropiada. 

Una sugerencia sería insertar flechitas para guiar a alguien leyendo la mapa, para que sepamos qué caja debemos leer, especialmente cuando hay más de una caja en la misma "linea" horizontal.

Otra sugerencia sería agregar un resumen de la entrevista con 4-5 puntos que te llamaron mucha atención, que podrías tener en mente cuando vayas a generar ideas en la etapa 2 esta semana.

Para la etapa 2, una sugerencia de mi parte sería hacer la actividad 8S (Ocho Loco) individualmente y luego usar SCAMPER (u otra estrategias en los videos) para ordenar las ideas y elegir una sola idea con que trabajarán en la etapa 3. (Es decir, repetir el proceso que practicamos durante las sesiones interactivas esta semana.)

¿Tienes preguntas para mi?

Adelante y mucha suerte,
Kasturi

**Reply by sowisla** [reply]
Muchas gracias por la retroalimentación, Kastury. Por cierto, tuve un contratiempo con la etapa 2, pero ya logré ponerme al corriente. Una disculpa por el retraso.

**Reply by KasturiSShah** [reply]
Hola @sowisla 
Gracias por el mensaje, espero todo bien. Me avisas cuando hayas subido la etapa 2 del proyecto, para que pueda revisarla. Pasaremos 2 semanas (semana pasada y semana presente) en el módulo 3 y empezaremos el módulo 4 después de la Semana Santa, entonces tienes tiempo para completar las etapas 😄 
Kasturi

**Reply by sowisla** [reply]
Muchas gracias, Kasturi. La etapa 2 ya la subí. También gracias por los avisos. Quedo atenta con lo restante del curso. Saludos. (:

**Comment by KasturiSShah** [comment]
Hola @sowisla 

Gracias por entregar la etapa 2 del proyecto final. La he revisado, y escribo con retrolamientación. En primer lugar, tus ideas son muy creativas y relevantes acerca del tema de tu proyecto. Tu video explica claramente el proceso 8S --> SCAMPER --> idea final, ¡felicidades! También te comento que no es necesario pedir disculpas por no prender la cámara, está perfectamente bien. 

¿Qué formato usarías para prototipar en la entrega 3? El prototip debe ser low fidelity, es decir, dibujos o diagramas. Algunas ideas sería una diagrama de la estructura del blog, indicando lugares donde estarán los vídeos, streaming, infografías, etc.

Sobre las dificultades que puedes encontrar al crecer la comunidad digital, ¿cómo fomentarás comunidad? Hay varias estrategias que grupos usan hoy en día, por ejemplo, correos semanales compartiendo tips sobre el mundo laboral, eventos como conferencias/paneles, grupos de trabajo en los cuales personas pueden colaborar para desarrollar skills aplicables en varios puestos profesionales de trabajo, etc. Quizás puedes investigar ejemplos que ya existen para pensar en ideas sobre cómo crecer la comunidad. 

¿Cómo andas con la entrega 3? Si necesitas una prórroga, me avisas.

Finalmente, veo varios alineamientos entre _¿Y ahora qué?_ y la iniciativa Patrones Hermosos, especialmente respecto al esfuerzo PH naciente que está trabajando para crear conexiones entre las alumnas y el mundo laboral. En algún momento, ¿sería posible discutir?

¿Tienes preguntas para mí? No hay prisa para contestar a este mensaje. Que disfrutes de la semana.

¡Feliz Semana Santa!

Saludos,
Kasturi

**Reply by sowisla** [reply]
Hola, Kasturi. Muchas gracias por tu retroalimentación. Sin duda me ayudaron para cumplir con la fase 3 del proyecto. Con gusto podemos hablarlo.

Saludos, 
Sofía.

**Comment by KasturiSShah** [comment]
Hola @sowisla 
Gracias por entregar la etapa 3 del proyecto final. También, gracias por el esfuerzo y tiempo que dedicaste a completar las etapas durante las semanas pasadas 😄  
Te escribo para avisar que la moderadora @Estefania-369 estará dándote retroalimentación para la etapa 3. Recibirás su retroalimentación por medio del hilo presente en el foro. 
Espero que descansaste durante el feriado.
Saludos a la distancia,
Kasturi

**Comment by Estefania-369** [comment]
Hola @sowisla ! :D

Gusto en saludarte, gracias por entregar la etapa 3 del proyecto final, así como tu dedicación en el desarrollo del mismo. Te escribo para compartir retroalimentación. 

Me parece excelente como estas desarrollando el proyecto y como lo explicas en el video, el objetivo general esta enfocado en lo que deseas lograr, mientras que los objetivos particulares están cubiertos y son claves para tener una base en el prototipado low-fidelity.

La pantalla principal del Blog, me agrada bastante, siento que si se lo presentas a más personas cumplirá con tu principal objetivo: que sea simple, fácil de entender. 
Es muy buena idea el carrete (carrusel slider) que presentas, así facilitas al usuario la búsqueda de información al estar clasificada. 

Has desarrollado muy buen trabajo, muestras como se vería el blog, dependiendo de lo que se seleccione, ya sea imagen, video, o texto, así, cuando llegue el momento de que recibas retro de más personas, se te facilitara saber que cambias con mas detalle, o si es simple para el usuario. 

De eso se trata, de identificar lo que requerimos agregar, cambiar, etc., del prototipo, las pestañas que comentas, como sugerencia, podrías agregar algunos botones, que dirijan a lo que seleccionemos, por ejemplo:

• Inicio
• Foro
• Blog
• Bolsa de empleo/ Vacantes
• Contacto
• Etc.
Dependiendo lo que consideres, y estes iterando. 

En la parte del Foro, me agrada como lo estas mostrando y la idea base cumple con el objetivo que deseas desarrollar.  También como sugerencia podrías agregar algunos lineamientos o reglas, para que los usuarios no se salgan de contexto. Por ejemplo: Centrarse en el tema. Respetar a los usuarios. No insultar ofender a los demás, etc. 
Otra sugerencia que podrías iterar de acuerdo a tus objetivos, es colocar a un lado del foro, una clasificación por temas, por ejemplo:

• Ofertas laborales o Vacantes 
• Consejos para el mundo laboral
• Experiencias
• Apoyo emocional 
• Quienes viven la problematica 
• Etc.

Esto lo puedes considerar, para una interacción entre los usuarios mas armónica y de acuerdo a los intereses de cada quien. 
Dependiendo de tus lineamientos, y lo que se vaya iterando, conforme vayas avanzando en el desarrollo del proyecto, podrás ir mejorándolo o adaptandolo si asi lo consideras. 
Me agradaria preguntar, ¿se va a requerir tener un registro en el blog, para tener acceso e interactuar en el Foro y con la comunidad?

¡Te felicito por tu trabajo! 👍 
Si tienes alguna duda, estamos para apoyarte.
Espero que estes teniendo una agradable semana, mucho éxito y mis mejores deseos.

Un abrazo
Estefania

**Reply by sowisla** [reply]
Muchas gracias por tu retroalimentación, Estefanía. Mencionaste varios elementos que no tenía contemplados y eso me ayudará.  Con el desglose que hiciste me diste muchas ideas nuevas. (: 

Respondiendo a tu pregunta, sí se tendría un registro para interactuar en el blog, esto con la finalidad de hacerlo seguro para todos los usuarios.

**Comment by sowisla** [comment]
Buenas noches. Espero que estén  muy bien, @KasturiSShah y @Estefania-369. Escribo para solicitar una prórroga para la entrega de la etapa 4 del proyecto; a más tardar el miércoles 19 de abril lo tendré listo. Agradezco su comprensión. 

Saludos,
Sofía.

**Reply by KasturiSShah** [reply]
Hola @sowisla 

Claro que sí, tienes una prórroga hasta el 19 de abril (miércoles). Mucha suerte, y avísanos  cualquier cosa.

Kasturi

**Reply by sowisla** [reply]
Muchas gracias, @KasturiSShah.

**Comment by Estefania-369** [comment]
@sowisla Hola bonita noche! 
Un gusto compartir contigo, me alegro mucho que te sirviera de algo la retroalimentacion. 
Perfecto, es bueno que consideres el registro para interactuar entre los usuarios.
Espero estes teniendo una buena semana 

Mucho exito! 
Un abrazo
Estefania

**Comment by Estefania-369** [comment]
Hola @sowisla :D

Me da gusto saludarte, gracias por entregar la etapa 4 del proyecto final, así como todo el trabajo en el desarrollo del mismo. Te escribo para compartir retroalimentación.

Son interesantes las observaciones que has recibido de la persona entrevistada para mejorar y seguir iterando tu prototipo. Me parece super bien tomar ideas de paginas que te inspiren para adaptarlas de acuerdo a los requerimientos de tu página. 

El botón que has agregado para regresar, facilita a los usuarios interactuar con la página, algunas cosas que has optado por eliminar, de acuerdo a las sugerencias recibidas por la persona entrevistada, me parece que le dan una mejor visión, es entendible y hace simple la navegación por tu blog. 

El registro, y solicitar un correo institucional da mayor seguridad y ofreces confianza a los usuarios.
Me encanta los temas que ofreces en el Blog, ya que es información de mucha utilidad para las personas egresadas, o que deseen desarrollarse en el mundo laboral. 

En la parte del Foro, me parece super que decidieras agregar a un lado, otros posibles temas en el que los usuarios se puedan sentir identificados y así puedan expresar sus ideas, sugerencias, experiencias, preguntas, etc., y recibir alguna orientación o compartir conocimiento entre otros usuarios, para apoyarse entre todxs. 

¡Es muy emocionante ver como has avanzado en el proyecto!
¡Te felicito por tu trabajo!
:+1: 
Ya estamos cerca de la clausura del curso, será muy emocionante ver tu trabajo junto con el de todas las chicxs. 
Si tienes alguna duda, estamos para apoyarte.
Espero que hayas teniendo una bonita semana, mucho éxito!
Un abrazo
Estefanía

**Reply by sowisla** [reply]
Muchas gracias por tu retroalimentación, Estefanía. Ya estoy alistando mi presentación final. Nos vemos pronto.

Saludos,
Sofía.

**Comment by Estefania-369** [comment]
Hola @sowisla ! :D 
Con gusto, muhas gracias a ti por compartir tu hermoso proyecto!
Me alegro de haber compartido este recorrido contigo, en donde aprendimos juntas. 
Me da gusto que estas alistando tu presetacion final. 

¡Sera emocionante que lo presentes en la clausura y  puedas compartir tu proyecto con todxs!
Se que seguirás mejorando, renovando hasta que logres incluso programarlo, desarrollar tu proyecto. 

Te deseo una bonita semana, y nos vemos pronto en la clausura, ¡que emoción!

Mucho éxito :+1: 
Un abrazo
Estefanía

**Comment by KasturiSShah** [comment]
Hola @sowisla buenas tardes
Ví que entregaste tu etapa 6 (¡gracias!). He movido el archivo, para que las etapas de cada equipo tienen el mismo formato. Aquí está el archivo que entregaste: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-07/etapa-6/etapa-6.md
Sería súper si puedes rellenar los otros puntos (tu nombre, descripción en una sola frase, etc).
Gracias y perdón la demora con crear el archivo para esta etapa (pensé que lo había hecho)
Felicidades otra vez por tu proyecto 😄 
Kasturi
